### ISSUE TITLE
💡[FEATURE]: Adding social media links

### ISSUE THREAD

**Issue by harinisayani** [akshitagupta15june/PetMe#1691]
**Is your feature request related to a problem? Please describe.**
I want to add more social media icons. 
**Describe the solution you'd like**
So, Everyone can communicate effectively.
**Describe alternatives you've considered**

**Additional context**
I am SSOC'24 contributor.
Please assign me this issue.

**Comment by satyaveni299** [comment]
Yes I can add social media links to the web page with icons ,can you plz assign me this under SSOC'24

**Comment by iamrudhresh** [comment]
I hope you're doing great! As a SSOC contributor, I'm keen on ramping up my involvement for SSOC'24. Could you please assign this issue to me? I'm excited to continue contributing to the project. Thanks a bunch!
